src/wp-admin/includes/plugin.php

```diff
@@ -1009,6 +1009,7 @@ function delete_plugins( $plugins, $deprecated = '' ) {
 			foreach ( $translations as $translation => $data ) {
 				$wp_filesystem->delete( WP_LANG_DIR . '/plugins/' . $plugin_slug . '-' . $translation . '.po' );
 				$wp_filesystem->delete( WP_LANG_DIR . '/plugins/' . $plugin_slug . '-' . $translation . '.mo' );
+				$wp_filesystem->delete( WP_LANG_DIR . '/plugins/' . $plugin_slug . '-' . $translation . '.mo.php' );
```

Comment:
What's the reason for using .mo.php instead of just .php?

Reply:
Glad you asked!

Turns out WP always tries to load `{$locale}.php` if it exists:

https://github.com/WordPress/wordpress-develop/blob/d088e31c73456179502c9bd5354fc43c6267bd7a/src/wp-settings.php#L519-L524

So this different file extension is needed to not break older sites.

See https://github.com/swissspidy/performant-translations/issues/77

Reply:
Ah! It's just odd to always keep the .mo in the name in case we want to add new formats.

Just thinking out load: What other formats in PHP we could use for the translations? Maybe a serialized string? Highly unlikely but still. An object? What about naming the current format `array.php` so we would have `de_DE.array.php` and maybe `de_DE.serialized.php` or `de_DE.object.php`?

Reply:
The thought behind `.mo.php` was that it makes it immediately obvious that it's related to translations, like `.l10n.php`. But if you want to split by format we can do `.array.php` too.

Reply:
I do have to say I'd prefer `.l10n.php` over `.array.php` as it feels weird to put the format into the file extensions, and I don't envision us using multiple different formats. And if we ever do, then we can just check the return value to know the type, we don't need the file extension for hat.

Reply:
A PR for `.l10n.php`: https://github.com/swissspidy/wordpress-develop/pull/50